src/components/global/Hero/styles.js

```diff
@@ -0,0 +1,71 @@
+export const heroStyles = {
```

Comment:
Could we rename the file to `heroStyles.js`, just for a bit more clarity? In fact, I wonder if we can simply move all this to `theme.js`.